### ISSUE TITLE
[BUG] nvstrings.strip does not strip unicode whitespace

### ISSUE THREAD

**Issue by lmeyerov** [rapidsai/cudf#3163]
**Describe the bug**

Running `myseries.str.strip()`  on a csv with unicode whitespace, e.g., `Zip` vs `Zip ` (csv `90 105 112  10  90 105 112 194 160  10`) fails to make the latter turn into the former. This caused some surprises in a data normalization flow. Note that when explicitly passing in the particular unicode whitespace as an explicit `strip` argument (`myseries.str.strip(' \t ')`), it worked as expected. See https://www.utf8-chartable.de/unicode-utf8-table.pl?utf8=dec&unicodeinhtml=dec for sample control characters. 

**Steps/Code to reproduce bug**
Follow this guide http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

`cudf.DataFrame(pd.DataFrame({'x': ['Zip', 'Zip ', 'Zip ']}))['x'].str.strip().len()`

**Expected behavior**
Expect sizes to be the same: `[3,3,3]`, not `[3,3,4]`

**Environment overview (please complete the following information)**
 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
 - Method of cuDF install: [conda, Docker, or from source]
   - If method of install is [Docker], provide `docker pull` & `docker run` commands used

**Environment details**
Please run and paste the output of the `cudf/print_env.sh` script here, to gather any other relevant environment details

`rapidsai/rapidsai-nightly:cuda10.0-runtime-ubuntu18.04`

**Comment by kkraus14** [comment]
Should resolve the bug in the new libcudf++ implementation.

**Comment by kkraus14** [comment]
Fixed as of 0.13:
```
In [103]: cudf.from_pandas(pd.DataFrame({'x': ['Zip', 'Zip ', 'Zip ']}))['x'].str.strip().str.len()
Out[103]:
0    3
1    3
2    3
Name: x, dtype: int32
```
